plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFunctionConfigProvider.java

```diff
@@ -23,15 +23,20 @@
 
     private FissionFunctionConfigProvider() {}
 
+    /**
+     * Fetetches env var 'FISSION_FUNCTION_BASE_URL' and returns it as a string, if it is not set will throw an error
+     *
+     * @return fissionFunctionbaseUrl, string prepresntation of the url url for fission endpoints
+     */
     public static String getFissionFunctionBaseURL() throws IOException
     {
-        if (this.fissionFunctionbaseUrl == null) {
+        if (fissionFunctionbaseUrl == null) {
```

Comment:
Noo! You've given me javadoc and taken away my precious this keywords! XD

Reply:
So that for some reason causes the linter to fail

Reply:
non-static variable this cannot be referenced from a static context

Reply:
Oh. Yeah. Didn't notice this was static. Surprised this even compiled!?

Reply:
I thin that was added on github so I don't think I ever compiled it with the this!

Reply:
Yeah I didn't realize the variable was static. I guess it's not quiiiiite a constant. This is fair. I was giving you bad advice!